### PR TITLE
Added Source for LC

### PR DESCRIPTION
**Pull request by appastair**
## Summary

Added Source/Scraper for Saint Lucia

## Additional notes

I noticed a tweet from [@SaintLuciaGov](https://twitter.com/SaintLuciaGov) regarding cases so I figured I'd add them. :hugs:

### CODE REVIEW

File: src/shared/sources/lc/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const assert = require('assert')
+
+const country = 'iso1:LC'
+
+module.exports = {
+  country,
+  timeseries: false,
+  friendly: {
+    name: "SAINT LUCIA'S COVID-19 DASHBOARD",
+    url: 'https://www.health.gov.au/'
```
Comment (camjc) [inline src/shared/sources/lc/index.js:10]: Guessing this was a copy-paste?

File: src/shared/sources/lc/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  },
+  scrapers: [
+    {
+      startDate: '2020-06-04',
+      crawl: [
+        {
+          type: 'page',
```
Comment (camjc) [inline src/shared/sources/lc/index.js:17]: I think page types need `data:` `table` or `paragraph` too

**Comment by camjc** [comment]
Looking good, couple-a comments, nice one @appastair .

File: src/shared/sources/lc/index.js (hunk 1 above)
Comment (appastair) [reply src/shared/sources/lc/index.js:10]: Ugh...yeah - sorry. This was my first scraper for 李 so... 😅

**Pull request merged**